apps/react/src/features/genres/pages/GenresPage/GenresPage.module.css

```
@@ -0,0 +1,23 @@
+.main {
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+	height: calc(100vh - var(--header-height));
+}
+
+.main__section {
+	min-width: 320px;
+	border: 1px var(--border-color) solid;
+}
+
+.main__section__list {
```

You can't have `BEM-element` of `BEM-element`
```suggestion
.main__section-list {
```